### ISSUE TITLE
[Bug] A "]" character at the end i

### ISSUE THREAD

**Issue by FahadBinHussain** [AmruthPillai/Reactive-Resume#1735]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Cloud (http://rxresu.me)

### Current Behavior

The Website field shows - `https://linktr.ee/_fahad`**]**

### Expected Behavior

The Website field should be - `https://linktr.ee/_fahad` 

### What browsers are you seeing the problem on?

Chrome

### What template are you using?

None

### Anything else?

![image](https://github.com/AmruthPillai/Reactive-Resume/assets/75292632/387e7d95-1e63-4ac6-afee-053f09befce1)

**Comment by AmruthPillai** [comment]
Can you check in the label field whether you have a `]` character in there?

![Screenshot 2024-01-25 at 9 43 27 AM](https://github.com/AmruthPillai/Reactive-Resume/assets/1134738/00f2763d-69d5-41ac-92e6-0802717a72c7)

**Comment by FahadBinHussain** [comment]
> Can you check in the label field whether you have a `]` character in there?
> 
> ![Screenshot 2024-01-25 at 9 43 27 AM](https://private-user-images.githubusercontent.com/1134738/299594653-00f2763d-69d5-41ac-92e6-0802717a72c7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyMzAyMzIsIm5iZiI6MTcwNjIyOTkzMiwicGF0aCI6Ii8xMTM0NzM4LzI5OTU5NDY1My0wMGYyNzYzZC02OWQ1LTQxYWMtOTJlNi0wODAyNzE3YTcyYzcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMjZUMDA0NTMyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmNjN2E5OTM4NjkxMDA4MmZlNWU2MGYwMzQzNGI3ZTdjM2FlZTdkYjVlMzkxYjFlMzczOGM4OTA2NjlhMDg2ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Eo29PnzyjJ1aw8HTYwIxz6Kc_-cnMN2nv9VOXa4OZ9g)

Yes, I also have "]" in there too. But what I am saying is that the "]" character automatically appears after importing data from the linkedin .zip.

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
